src/Adapters/Email/SimpleMailgunAdapter.js

```diff
@@ -0,0 +1,39 @@
+import Mailgun from 'mailgun-js';
+
+let SimpleMailgunAdapter = mailgunOptions => {
```

Comment:
Actually, you still need to return an Object in the end, I'm not sure of the performance advantage of returning the object as you do instead of doing

```
export class SimpleMailgunAdapter {
constructor(options) {
this.mailgun = Mailgun(mailgunOptions);
}
sendMail(to, subject, text) {}
sendVerificationEmail({ link, user, appName, }) {}
}
```